packages/web-components/src/tabs/tab-list/tab-list.ts

```diff
@@ -0,0 +1,82 @@
+import { attr, booleanConverter } from '@microsoft/fast-element';
+import { FASTTabs } from '@microsoft/fast-foundation';
+
+export interface TabData {
+  id: string;
+  x: number;
+  y: number;
+  height: number;
+  width: number;
+}
+/**
+ * TabList extends FASTTabs and is used for constructing a fluent-tab-list custom html element.
+ *
+ * @class TabList component
+ * @public
+ */
+export class TabList extends FASTTabs {
+  @attr appearance?: 'subtle' | 'transparent';
+
+  @attr({ converter: booleanConverter })
+  disabled?: boolean;
+
+  @attr size?: 'small' | 'medium' | 'large' = 'medium';
```

Comment:
For all typings in here, please enumerate that they are public, protected, or private. All attrs will be public